samples/bsmTx.ini

```diff
@@ -0,0 +1,133 @@
+[_settings]
```

Comment:
We can remove this settings block - no longer needed as these arguments are test-harness specific and passed in by command line.